KlingonAssistant/data/mem-17-tlh.xml

```diff
@@ -804,17 +804,17 @@ There is an idiom, {Hem; tlhIngan rur}.</column>
       <column name="_id">33701</column>
       <column name="entry_name">tlhIngan Hol</column>
       <column name="part_of_speech">n:deriv</column>
-      <column name="definition">Klingon (language)</column>
+      <column name="definition">Klingon language</column>
```

Comment:
Note that this bracketing parallels other pairs where the name of the person is identical in English to the name of the language or planet.